atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransactionManager.java

```diff
@@ -273,9 +279,30 @@ public static TransactionManager create(MetricsManager metricsManager,
                 : transactionManager;
     }
 
+
+    public static SerializableTransactionManager createForTest(MetricsManager metricsManager,
```

Comment:
Would rather avoid additional constructors if we don't strongly need them. In this case, the caller can just use the other constructor and list the timestamp service twice.

Reply:
That was my initial approach, will revert to it. I introduced this to minimise the diff, but it's not used in many places so happy to make invocations a bit more verbose.